src/registrar/admin.py

```diff
@@ -222,11 +256,97 @@ class ContactAdmin(ListHeaderAdmin):
 
     search_fields = ["email", "first_name", "last_name"]
     search_help_text = "Search by firstname, lastname or email."
+    list_display = [
+        "contact",
+        "email",
+    ]
+
+    # We name the custom prop 'contact' because linter
+    # is not allowing a short_description attr on it
+    # This gets around the linter limitation, for now.
+    def contact(self, obj):
+        if obj.first_name or obj.last_name:
+            return obj.get_formatted_name()
+        elif obj.email:
+            return obj.email
+        elif obj.pk:
```

Comment:
Note that this will not execute if obj.pk == 0. For integers, a truthy/falsy shorthand will return false if that integer is equal to 0. Here is a good resource which explains this in depth:
https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/

Reply:
2 notes:
- The DB API uses a 'bool(pk_val) == false' check to determine if a model instance is a new item needs to be INSERTed, or an existing entry that needs to be UPDATEd.
Most databases use PK's starting at 1, but if you try to manually use a pk value of 0, the object cannot be saved - it can only be recreated.

- This code is a duplicate of the _str_ method in the contact model

Reply:
@rachidatecs Ah, I was not aware that this is not a possible state for Django. Lack of knowledge on my part, I think this is a non issue then.